src/librbd/mirror/snapshot/GroupUnlinkPeerRequest.cc

```diff
@@ -203,9 +207,42 @@ void GroupUnlinkPeerRequest<I>::handle_remove_peer_uuid(int r) {
 
 template <typename I>
 void GroupUnlinkPeerRequest<I>::remove_group_snapshot(
-                              cls::rbd::GroupSnapshot group_snap) {
+                              cls::rbd::GroupSnapshot group_snap,
+                              bool is_snap_incomplete) {
```

Comment:
Is this argument required? The function can just check group_snap.state directly.

Reply:
You could also just update the group_snap with the snap id of each found image snap and let the existing code proceed to perform the deletes. But this fine too.

Reply:
> Is this argument required? The function can just check group_snap.state directly

Any comments?

Reply:
I prefer passing the arg instead of redoing the same again the loop in the remove_group_snapshot()

Reply:
Ack.